src/gluonts/shell/serve.py

```diff
@@ -101,37 +102,99 @@ def load_config(self) -> None:
             if key in self.cfg.settings and value is not None:
                 self.cfg.set(key, value)
 
-    def load(self) -> flask.Flask:
+    def load(self) -> Flask:
         return self.application
 
     def stop(self, *args, **kwargs):
         self.application.logger.info('Shutting down GluonTS scoring service')
 
 
+def make_flask_app(predictor_factory, execution_params) -> Flask:
+    app = Flask('GluonTS scoring service')
+
+    @app.route('/ping')
+    def ping() -> Response:
+        return Response(status=200)
```

Comment:
Why return the verbose response?

Reply:
The PyCharm linter was complaining due to the type mismatch. Reverted.